packages/next/lib/chalk.ts

```diff
@@ -0,0 +1,9 @@
+let chalk: typeof import('next/dist/compiled/chalk')
+
+if (!process.browser) {
```

Comment:
```suggestion
if (typeof window === 'undefined') {
```
Should we use `typeof window` for the browser checks instead of `process.browser` since that's what we usually recommend for users?

Reply:
Sounds good! Will the `typeof window === 'undefined'` branch be stripped off by webpack, when we bundle this into a function?

Reply:
After checking with Sokra — we need to add that to the define plugin for dead code elimination.

Reply:
`typeof window` is replaced by babel and swc currently, [related config](https://github.com/vercel/next.js/blob/c879fce0d465cd96d0d05e329c0a8a83f46e29d9/packages/next/build/swc/options.js#L57-L62)

Reply:
Wow, thanks!